semgrep.yml

```diff
@@ -0,0 +1,11 @@
+rules:
+  - id: code-block-python
+    paths:
+      include:
+        - "python/ray/data/**/*.py"
+        - "doc/source/data/**/*.rst"
+    languages:
+      - generic
+    message: "Don't use 'code-block:: python', it's not tested! Use 'testcode' instead!"
```

Comment:
Will people understand what to do with this error message? Are there links to documentation?

Reply:
Ah, good call. I've added a link to the docs.